backend-peer/src/lib/services/cloudagent.ts

```diff
@@ -71,17 +74,15 @@ export default class CloudagentManager {
 
     const responseBody = await res.json()
 
-    if (res.ok) {
-      return responseBody as DidResolutionResultProps
+    if (!res.ok) {
+      throw new ServiceUnavailable('Error fetching cloud agent')
```

Comment:
so we don't want to include the response body anymore in the error message?

Reply:
No, from a security perspective this type of internal information should never be sent as an API response. Instead, this should be stored in the logs.